cli/src/main/java/org/aya/cli/parse/AyaProducer.java

```diff
@@ -270,7 +270,8 @@ public QualifiedID visitQualifiedId(AyaParser.QualifiedIdContext ctx) {
       throw new ParsingInterruptedException();
     }
     var string = ctx.STRING();
-    if (string != null) return new Expr.LitStringExpr(pos, string.getText());
+    if (string != null)
+      return new Expr.LitStringExpr(pos, string.getText().substring(1, string.getText().length() - 1));
```

Comment:
Escape

Reply:
I'm not sure what this means. Should I do `translateEscapes()` here?

Reply:
![image](https://user-images.githubusercontent.com/16398479/168959496-92c97cfd-bb2f-4a18-9e9c-5550624b75b8.png)